pkgs/tools/networking/networkmanager/default.nix

```diff
@@ -61,7 +64,8 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-nP/SrcaGUTFt8tL4oJ4XF7sdDC6jic/HIaAQnbmzWCY=";
   };
 
-  outputs = [ "out" "dev" "devdoc" "man" "doc" ];
+  outputs = [ "out" "dev" ]
+    ++ lib.optionals (stdenv.buildPlatform == stdenv.hostPlatform) [ "devdoc" "man" "doc" ];
```

Comment:
Are the man pages not cross buildable either?

Reply:
This patch will be needed https://github.com/void-linux/void-packages/blob/master/srcpkgs/NetworkManager/patches/no-gtk-doc.patch

Though I can just copy the outputs from the native package to the new one